src/main/java/Ui.java

```diff
@@ -135,6 +135,11 @@ public void showTaskDeleted(Task task, TaskList taskList) {
         printLine();
     }
 
+    /**
+     * Prints the list of tasks which has descriptions
+     * matching that of a target String.
+     * @param taskList List of matching tasks
+     */
     public void showFoundTasks(LinkedList<Task> taskList) {
         ListIterator<Task> iter = taskList.listIterator();
         Task current;
```

Comment:
Your code looks pretty clean with the accurate variable naming as well as javadocs comments. Perhaps you could consider making packages to make it more organized? Otherwise it looks good